### DISCUSSION TITLE
Lighting patterns are different from client to client?

### DISCUSSION THREAD

**Discussion by Tom718-uk** [matsn0w/IIS-EmergencyLS-ELS-FiveM#129] [Q&A] [answered]
Why is another players ELS patter slower/different to what it is on my client side?

I added in a sprinter van. For me the lights look normal. For him, they are very slow, and often only lighting one side of the lightbar...

Not sure why.

https://user-images.githubusercontent.com/66221172/201531819-d1ef9310-7b71-4fc5-9427-8d8d95bd273c.mp4

**Comment by matsn0w** [comment, marked as answer]
Hey, thanks for posting! This is a known issue. Please see #41.

**Reply by Tom718-uk** [reply]
Hey there! This is bearable. We have come across another problem. Whenever you play a siren on a vehicle, it activates on all vehicles. The lighting patterns we can deal with.

Originally we had serverside sirens with wmsirens however, thinking that was the issue we removed it. But the vanilla sirens are still having the same issue. The lights are controlled by each individual car, but the sirens seem to go on every car, no matter what.

Any ideas on how to fix this?

**Reply by matsn0w** [reply]
Ok, are you using OneSync? Becuase that isn't supported at this time...

**Reply by Tom718-uk** [reply]
yes, If we turn off onesync, what effect will that have on the server?

**Reply by Tom718-uk** [reply]
or, how long would it take to get an update where onesync is compatible?

**Reply by matsn0w** [reply]
> yes, If we turn off onesync, what effect will that have on the server?

Probably a lot. As a rule of thumb, you can't use non-OneSync plugins in a OneSync server and vice versa.

> or, how long would it take to get an update where onesync is compatible?

Can't give you an ETA on that. I am planning to support this some day, but I am not making any promises ;)

**Reply by Tom718-uk** [reply]
Is there any ELS system out there that is compatible with one-sync?

**Reply by matsn0w** [reply]
Not that I know of, otherwise I'd probably lend some features ;)

**Reply by Tom718-uk** [reply]
What would be your recommendation for us then? We have a lot of ELS vehicles we want to use.

**Reply by matsn0w** [reply]
Difficult. I have no experience with OneSync. It highly depends on what scripts you want to use. If ELS is important to you, I'd probably go without it entirely. But that means you cannot use any OneSync script anymore. The same applies if you want to stick to OS.
